scripts/screener/screener.runner.ts

```diff
@@ -46,7 +46,7 @@ async function scheduleScreenerBuild(
     pullRequest: buildInfo.pullRequest,
   };
 
-  const response = await fetch(environment.screener.proxyUri.replace('ci', 'runner'), {
+  const response = await fetch(`${environment.screener.proxyUri}/api/runner`, {
```

Comment:
added for testing purposes, will be removed when merging and added in a follow up PR as stated in a previous comment